fe/fe-core/src/main/java/com/starrocks/credential/aws/AWSCloudCredential.java

```diff
@@ -200,20 +210,20 @@ private AwsCredentialsProvider getBaseAWSCredentialsProvider(boolean useAWSSDKDe
     }
 
     private void applyAssumeRole(String baseCredentialsProvider, Configuration configuration) {
-        configuration.set("fs.s3a.assumed.role.credentials.provider", baseCredentialsProvider);
+        configuration.set(Constants.ASSUMED_ROLE_CREDENTIALS_PROVIDER, baseCredentialsProvider);
```

Comment:
What's the value "Constants.ASSUMED_ROLE_CREDENTIALS_PROVIDER" in hadoop 3.4.0? Is it different from "fs.s3a.assumed.role.credentials.provider" ?

Reply:
the same

Reply:
This way of writing will be able to find problems when compiling in the future